SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -19,13 +22,13 @@ eval "${ACTIVATE_ENV_CMD}"
 
 # Install packages in the environment and the same ipykernel that the Jupyter server uses
 _log "Installing packages from ${REQ_PATH}..."
-pip install -r "${REQ_PATH}" "ipykernel==${IPYKERNEL_VERSION}" ${NXCALS} ${SPARKMONITOR} ${SWANPORTALLOCATOR} 2>&1 | tee -a ${LOG_FILE} # TODO
+uv pip install -i https://acc-py-repo.cern.ch/repository/vr-py-releases/simple --allow-insecure-host acc-py-repo.cern.ch -r "${REQ_PATH}" "ipykernel==${IPYKERNEL_VERSION}" ${NXCALS} ${SPARKMONITOR} ${SWANPORTALLOCATOR} 2>&1 | tee -a ${LOG_FILE} # TODO
```

Comment:
Can you double check with Philip that there is no other way to configure this? Can you let him know about the issue you had when picking this repository information from the configuration file?